pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -506,10 +508,16 @@ func generateAndSendImportSpans(
 					if err != nil {
 						return err
 					}
-					break
+
+					if key.Compare(span.EndKey) < 0 {
+						key = span.EndKey
+					} else {
+						break
+					}
+				} else {
+					key = startEndKeyIt.value()
 				}
 
-				key := startEndKeyIt.value()
 				if span.Key.Compare(key) >= 0 {
```

Comment:
only ends key is exclusive, right, not both/the whole span? and end being exclusive is the norm across the codebase so I think we don't actually want to call it out as special, since that could just be confusing when that's the norm (e.g. reader wonder if it is special/different).

Reply:
good point. I take my suggestion back.